### PR TITLE
[RFR] One push per review

### LINKED ISSUE #59: Make multiple fixes in one commit (for review comment)
Because the GitHub API takes ~30 secondes to digest a commit.
Hence, the update reference doesn't work.

See https://github.com/prose/prose/issues/148

image

### PR DESCRIPTION
**Pull request by Kmaschta**
Fix #59

### CODE REVIEW

File: sedy/src/parser/parsePullRequestReview.js [hunk 1]
```diff
@@ -1,43 +1,45 @@
...
-                position: comment.position,
-                url: comment.html_url,
-            },
-            commit: {
-                id: comment.commit_id,
-            },
-            matches: getSedyCommandsFromComment(comment.body),
-        }));
+    const fixes = comments.map(comment => ({
+        // @TODO Split comment part into mutliple ones: diff, action, sender, etc
```
Comment (djhi) [inline sedy/src/parser/parsePullRequestReview.js]: still relevant ?

**Comment by Kmaschta** [comment]
Please do not merge, we have a loop on fixes in the index

**Comment by Kmaschta** [comment]
Good for me now!

File: sedy/src/answerer.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export default (githubClient, parsedContent) => {
```
Comment (fzaninotto) [inline sedy/src/answerer.js]: Not fan of the nem, you should try something more specific, like "CodeReviewCommentAnswerer"

File: sedy/src/commiter.js [hunk 3]
```diff
@@ -1,85 +1,41 @@
...
-            for (const fix of fixedContent) {
-                const commit = yield digestCommit(parsedContent, fix);
-                commits.push(commit);
-            }
-
-            logger.info('Successful commits', { commitsIds: commits.map(commit => commit.sha) });
+    const push = function* (lastCommitSha) {
+        if (lastCommitSha) {
+            yield git.push(parsedContent.pullRequest.ref);
+            logger.info('Successful pushed the branch', { branch: parsedContent.pullRequest.ref });
```
Comment (fzaninotto) [inline sedy/src/commiter.js]: s/Successful/Successfully/

**Pull request merged**